io.github.bothlab.syntalos.yaml

```diff
@@ -0,0 +1,262 @@
+id: io.github.bothlab.syntalos
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: syntalos
+rename-icon: syntalos
+rename-desktop-file: syntalos.desktop
+
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=all
+  - --filesystem=host
```

Comment:
this is a wide filesystem permission.

Reply:
Yes, this is required because Syntalos will actually need to probe the host system for available external applications, available udev rules for new hardware to notify the user in case anything is missing on the host etc.
The user may likewise want to store generated data in a number of places. For our lab it's usually some drive mounted in `/mnt`, but I have also seen other labs which had a dedicated `/scratch` root directory.

Reply:
and how is it gonna launch these external applications? etc.

Reply:
There's `flatpak-spawn --host` for that :-)

Reply:
ah yeah with the black hole size dbus hole you put above.

Reply:
Exactly. Not great, but workable. This is software run in specialized environments anyway, where people need to tinker with it. That's why in this particular instance I don't see it as much of an issue as it would be with a less developer-focused tool like a mediaplayer.
So far, the software was only deployed in source or as a local deb package.